README.md

```diff
@@ -15,27 +15,27 @@ There will be support for 5 different filesystems:
 - zfs
 - bcachefs
 
-There will be also automatic snapshotting implemented, to make it easy for users to rollback if they run into issues.
-bcachefs could be generally a good idea for the Handhelds, but it is not fully ready yet and we don't suggest it for now.
+There will also be automatic snapshotting implemented, to make it easy for users to rollback if they run into issues.
+Bcachefs could be generally a good idea for handhelds, but it is not fully ready yet and we don't suggest it for now.
 
 ## Features
 - scx_lavd used as default scheduler - LAVD is a latency sensitive scheduler, which is intended to be used for handhelds and gaming
 - Steam Deck like Steam Expierence
-- Gamescope Session proper implemented 
+- Gamescope Session properly implemented 
 - Kernel Patched to have support for Steam Deck OLED
 - Steam Deck OLED Firmware included as default
 - HDR correctly implemented via kerel patches
 - All dependecies and packages bundled for a proper gaming expierence (cachyos-gaming-meta)
 - Support for Winesync/Fastsync/NTSync (proton-cachyos provides a fastsync patched proton)
 - BBRv3 used as default
 - OpenRGB Patches included
-- Screensharing for Discord (via xwaylandvideobridge)
+- Screensharing for Discord (via xwaylandvideobridge) (currently broken on versions past 0.0.43 so you have to downgrade to 0.0.43 [click here for more information](https://github.com/flathub/com.discordapp.Discord/issues/380))
```

Comment:
Archlinux is on 0.0.49, so this doesnt affect. Were using native packages and not flatpak

Reply:
Please remove it anyways or point to vesktop, since it works here even with the default electron without xwayland videobridge.